### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by jacobrosenthaldfci**
This fixes two bugs:
* error when using Boolean labels (#193)
* error in SlideData.__repr__() when it was loaded directly from .h5path (#192)

### CODE REVIEW

File: pathml/core/slide_data.py [hunk 1]
```diff
@@ -336,7 +338,12 @@ def shape(self):
         Returns:
             Tuple[int, int]: Shape of image (H, W)
         """
-        return self.slide.get_image_shape()
+        if self.backend == "h5path":
```
Comment (ryanccarelli) [inline pathml/core/slide_data.py:341]: We could add the image shape to our .h5path specification?

**Review by ryanccarelli — CHANGES_REQUESTED** [review]

File: pathml/core/slide_data.py (hunk 1 above)
Comment (jacobrosenthaldfci) [reply pathml/core/slide_data.py:340]: added

File: tests/core_tests/test_slide_data.py [hunk 2]
```diff
@@ -126,7 +126,11 @@ def test_read_write_heslide(tmp_path, example_slide_data_with_tiles):
     path = tmp_path / "testhe.h5"
     slidedata.write(path)
     readslidedata = SlideData(path)
+    repr(readslidedata)
     assert readslidedata.name == slidedata.name
+    print(readslidedata.shape)
+    print(type(readslidedata.shape))
```
Comment (ryanccarelli) [inline tests/core_tests/test_slide_data.py:132]: remove print statements

**Review by ryanccarelli — CHANGES_REQUESTED** [review]

File: tests/core_tests/test_slide_data.py (hunk 2 above)
Comment (jacobrosenthaldfci) [reply tests/core_tests/test_slide_data.py]: done

**Pull request merged**